ocs_ci/ocs/bucket_utils.py

```diff
@@ -2974,6 +2980,8 @@ def upload_obj_versions(mcg_obj, awscli_pod, bucket_name, obj_key, amount=1, siz
             out_yaml_format=False,
         )
 
+    return etags
```

Comment:
It seems like we are returning empty list since code is not adding any entries in etags

Reply:
Correct - it's due to a mistake in my merge conflict resolution for when I rebased this PR off of #11322. Will fix in a follow-up PR.

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/11438/commits/0eda7a886d95a7d462b578683e127fc71500d443